src/components/UserMenu/index.js

```diff
@@ -1,5 +1,6 @@
 import { signInMethods } from '../../firebase/config';
 import useAuth from '../../hooks/useAuth';
+import React from 'react';
```

Comment:
idem

Reply:
Solução para o erro `'React' must be in scope when using JSX  react/react-in-jsx-scope`.
Vou verificar atualizações para ver se a exigência persiste.